content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -521,6 +521,14 @@ The {{< glossary_tooltip text="control plane" term_id="control-plane" >}} adds t
 
 If the number of backend endpoints falls below 1000, the control plane removes this annotation.
 
+### endpoints.kubernetes.io/last-change-trigger-time
+
+Example: EndpointsLastChangeTriggerTime = "endpoints.kubernetes.io/last-change-trigger-time"
+ 
+Used on: Endpoints
+
+EndpointsLastChangeTriggerTime is the annotation key set to an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object that represents the timestamp (stored as RFC 3339 date-time string, e.g. '2018-10-22T19:32:52.1Z') of the last change, of some Pod or Service object, that triggered the endpoints object change. 
```

Comment:
`EndpointsLastChangeTriggerTime` is an implementation detail. We should document the API rather than the [k/k](https://github.com/kubernetes/kubernetes) implementation.

Reply:
@sftim i have corrected description and formating